user/serializers.py

```diff
@@ -22,3 +22,36 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.EmailField(label="Email", write_only=True)
+    password = serializers.CharField(
+        label="Password",
+        style={"input_type": "password"},
+        trim_whitespace=False,
+        write_only=True,
+    )
+
+    token = serializers.CharField(label="Token", read_only=True)
+
+    def validate(self, attrs):
+        email = attrs.get("email")
+        password = attrs.get("password")
+
+        if email and password:
+            user = authenticate(
+                request=self.context.get("request"),
+                email=email,
+                password=password,
+            )
+
+            if not user:
+                msg = "Wrong credentials"
+                raise serializers.ValidationError(msg, code="authorization")
```

Comment:
Consider providing a more descriptive error message for failed authentication attempts, such as 'Unable to authenticate with provided credentials.' This can help users understand the issue more clearly.